ow_plexil/src/plans/TestAntennaMovement.plp

```diff
@@ -0,0 +1,42 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Simple plan to test Pan and Tilt functions
+
+#include "plan-interface.h"
+
+LibraryAction TestPanTilt (In Real Goal, In Real Action, InOut Integer TestSuccess);
+
+TestAntennaMovement: CheckedSequence
+{
+  Integer TestCount = 0;
+
+  //initial location
+  LibraryCall Pan(Degrees = 0);
+  LibraryCall Tilt(Degrees = 0);
```

Comment:
Suggest reversing these, since pan can be potentially obstructed by the current tilt angle, but tilt to 0 should never be obstructed.

Reply:
I think these lines were added to set an known pose for the antenna before performing the test sequence. Otherwise, the test may or may not succeed based on the current antenna pose before running the test.

Reply:
> I think these lines were added to set an known pose for the antenna before performing the test sequence. Otherwise, the test may or may not succeed based on the current antenna pose before running the test.

I knew that.  Was just suggesting to do tilt before pan.  Minor point actually, since the antenna would have to be pretty toppled over to block a pan.